spk/hassio/src/service-setup.sh

```diff
@@ -0,0 +1,93 @@
+CFG_FILE="/usr/local/${SYNOPKG_PKGNAME}/etc/hassio.json"
+
+service_postinst() {
+    if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ]; then
+        DATA_DIR="${wizard_share_path}/${wizard_folder_name}"
+        # Fix install directory
+        mkdir -p "${DATA_DIR}"
+
+        HOMEASSISTANT_DOCKER="homeassistant/qemux86-64-homeassistant"
+        HASSIO_DOCKER="homeassistant/amd64-hassio-supervisor"
+
+        # Read infos from web
+        URL_VERSION="https://s3.amazonaws.com/hassio-version/stable.json"
+        HASSIO_VERSION=$(curl -s $URL_VERSION | jq -e -r '.supervisor')
```

Comment:
If this config was pulled at SPK build time as a dependency of building the SPK, it could be interpreted for a constant version (see `spk/hassio/Makefile` line 2 discussion).  I'm not sure it's the best method, but it's more consistent than `$(date +...)` which sets a different version on the same thing depending on when it's done.

I'm not sure whether the package should display the Supervisor rev (currently 192) or the homeassistant rev (currently 0.102.3) based on what a user thinks they're seeing.

Reply:
The spk-package is just a template on building the hassio docker environment. So we should not bother so much with this.

Reply:
Agreed the spk-package version should be an independent running number from supervisor or homeassistant as it pulls the latest versions of those as of install. Might be a good idea to give the user an idea what the latest versions of those where at time of spk release, but I'd consider that a documentation thing.